docs/UserGuide.md

```diff
@@ -107,16 +132,17 @@ Format: `add_class tn/TUTORIAL_NAME v/VENUE d/DAY tm/TIME wk/WEEK`
 * `WEEK` refers to the number of weeks the class will be held for
 
 Examples:
-* `add_class tn/T04 v/LT13 d/Monday tm/13:00`
-* `add_class tn/G04 v/E-LEARNING d/Wed tm/10:00`
+* `add_class tn/T04 v/LT13 d/Monday tm/13:00 wk/13`
+* `add_class tn/G04 v/E-LEARNING d/Wed tm/10:00 wk/13`
 
 #### Adding student to a class: `add_student`
 
 Adds a specified student to a given class.
 
 Format: `add_student n/NAME id/STUDENT_ID tn/TUTORIAL_NAME`
 
-* Adds the person with the specified `NAME` as a student belonging to a specified class.
+* Update the person with the specified `NAME` to a `Student` contact and add the student to the specified class.
```

Comment:
I think we should mention the id field too!

Reply:
updated!